### ISSUE TITLE
linkages between Arctos occurrence records appear broken

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/globalbioticinteractions#461]
In the recent enhancements of GloBI, it appears that the Arctos links are no longer being established via the associatedOccurrences. 

E.g., http://arctos.database.museum/guid/DMNS:Mamm:11005 has a link to http://arctos.database.museum/guid/DMNS:Para:908 , however, GloBI appears to know nothing about it (see [search page](https://www.globalbioticinteractions.org/?accordingTo=http%3A%2F%2Farctos.database.museum%2Fguid%2FDMNS%3AMamm%3A11005&interactionType=interactsWith) ) .

See attached screenshots . 
![Screenshot from 2020-02-21 14-26-38](https://user-images.githubusercontent.com/1084872/75076945-3cdb2f80-54b6-11ea-80e2-541fe248ddaf.png)
![Screenshot from 2020-02-21 14-26-30](https://user-images.githubusercontent.com/1084872/75076948-3e0c5c80-54b6-11ea-9e2c-e39384453670.png)

**Comment by jhpoelen** [comment]
It appears that the linked records are no longer being discovered because of a configuration issue. 

last time vertnet was indexed as expected was on:
```
dexing archive(s) from [file:/home/elton/datasets/globalbioticinteracti
ons/vertnet/88564467d633aba7b98efa5eb75e4bd88a148d08570947b68cf9050905936851]...
2020-02-10 13:40:33,520 [main] INFO  org.eol.globi.data.StudyImporterForRSS - in
dexing archive(s) from [file:/home/elton/datasets/globalbioticinteracti
ons/vertnet/88564467d633aba7b98efa5eb75e4bd88a148d08570947b68cf9050905936851] do
ne: indexed [60617] occurrences
...
```
see also https://github.com/globalbioticinteractions/vertnet/commit/8bafd31e655e532146095f782889ec61b06634eb#r37430409 .

**Comment by jhpoelen** [comment]
GloBI search index was reverted to version 2020-02-10 to re-establish the record linkages. Also, spot checks were made to verify the correct linkages.

Also, a fix was implemented and released in v0.16.1 . If all goes well, in a couple of days, the changes will propagate and the index for https://globalbioticinteractions.org will be updated.

![Screenshot from 2020-02-21 17-02-50](https://user-images.githubusercontent.com/1084872/75083201-525b5400-54cc-11ea-9d76-5b9d8f1d2fe6.png)
![Screenshot from 2020-02-21 17-02-43](https://user-images.githubusercontent.com/1084872/75083203-538c8100-54cc-11ea-932e-504d8c68e1dc.png)
